iocore/net/SSLNextProtocolAccept.cc

```diff
@@ -77,14 +77,20 @@ struct SSLNextProtocolTrampoline : public Continuation {
 
     vio   = static_cast<VIO *>(edata);
     netvc = dynamic_cast<SSLNetVConnection *>(vio->vc_server);
-    ink_assert(netvc != nullptr);
 
     switch (event) {
     case VC_EVENT_EOS:
     case VC_EVENT_ERROR:
     case VC_EVENT_ACTIVE_TIMEOUT:
     case VC_EVENT_INACTIVITY_TIMEOUT:
-      netvc->do_io_close();
+      if (netvc != nullptr) {
+        netvc->do_io_close();
+      } else { // Try making a unix netvc
```

Comment:
`do_io_close()` is a virtual function, so I think we can simply call `vio->vc_server->do_io_close()` without casting.